packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AdvancedCoinSettingsModal/CustomBackends/CustomBackends.tsx

```diff
@@ -107,6 +107,7 @@ export const CustomBackends = ({ network, onCancel }: CustomBackendsProps) => {
     const { defaultUrls, isLoading } = useDefaultUrls(coin);
     const editable = type !== 'default';
     const { ref: inputRef, ...inputField } = register(name, { validate });
+    const isSubmitButtonDisabled = type !== 'default' && !!error;
```

Comment:
Nit: could use `editable` for consistency, because `editable` is also used for conditions to display the inputs below.
```suggestion
    const isSubmitButtonDisabled = editable && !!error;
```

Reply:
good point, done